LICENSE

```diff
@@ -1,6 +1,10 @@
 MIT License
 
-Copyright (c) 2021 David Widmann
+Copyright (c) 2021 Brian J Smith, David Widmann and contributors:
```

Comment:
Maybe consider adding the Turing team here - there are a lot of improvements since the code is extracted from `Mamba.jl`.